docs/guide/node_in_cloud.md

```diff
@@ -0,0 +1,152 @@
+# Deploy tfjs-node project on cloud platform
+
+This doc describes how to run a Node.js process with @tensorflow/tfjs-node package on cloud platforms.
+
+Starting from tfjs-node@1.2.4, running Node.js project on cloud platforms does not require extra configurations. This guide will use the [mnist-node](https://github.com/tensorflow/tfjs-examples/tree/master/mnist-node) example in @tensorflow/tfjs-examples repository, which trains MNIST with tfjs-node package on Heroku and GCloud. The details of Heroku’s Node.js Support can be found in [Heroku Node.js Support](https://devcenter.heroku.com/articles/nodejs-support) article. The details of Node.js on Google Cloud Platform can be found on the [Google Cloud page](https://cloud.google.com/nodejs/docs/).
+
+## Deploy Node.js project on Heroku
+
+### Prerequisites
+
+1. Node.js and npm installed
+2. Heroku account
+3. Heroku CLI
+
+### Create the Node.js app
+
+1. Create a folder and copy the `data.js`, `main.js`, `model.js` and `package.json` files from the [mnist-node](https://github.com/tensorflow/tfjs-examples/tree/master/mnist-node) example.
+2. Make sure the @tensorflow/tfjs-node dependency is @1.2.4 or newer version.
+
+### Build your app and run it locally
+
+1. Run the `npm install` command in your local directory to install the dependencies that are declared in the `package.json` file. You should be able to see the tfjs-node package is installed and libtensorflow is downloaded.
+
+```
+$ npm install
+> @tensorflow/tfjs-node@1.2.5 install mnist-node/node_modules/@tensorflow/tfjs-node
+> node scripts/install.js
+
+CPU-linux-1.2.5.tar.gz
+* Downloading libtensorflow
+[==============================] 22675984/bps 100% 0.0s
+* Building TensorFlow Node.js bindings
+```
+
+2. Train the model locally by running `npm start`.
+
+```
+$ npm start
+> tfjs-examples-mnist-node@0.1.0 start /mnist-node
+> node main.js
+
+2019-07-30 17:33:34.109195: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
+2019-07-30 17:33:34.147880: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3492175000 Hz
+2019-07-30 17:33:34.149030: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x52f7090 executing computations on platform Host. Devices:
+2019-07-30 17:33:34.149057: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
+
+Total params: 594922
+Trainable params: 594922
+Non-trainable params: 0
+_________________________________________________________________
+Epoch 1 / 20
+Epoch 1 / 20
+========================>----------------------------------------------------------------------------------: 35.5
+```
+
+3. Make sure you ignore build artifacts, such as node_modules, in your .gitignore file.
+
+### Create and deploy the heroku app
+
+1. Create a new app on heroku website
```

Comment:
"Create a new app on the Heroku website".

Reply:
Done.